libs/core/langchain_core/tools/convert.py

```diff
@@ -42,6 +48,14 @@ def tool(
         error_on_invalid_docstring: if ``parse_docstring`` is provided, configure
             whether to raise ValueError on invalid Google Style docstrings.
             Defaults to True.
+        handle_tool_error: Handle the content of the ToolException thrown. If False
+              do nothing, If True returns 'Tool execution error'. If string then
+              returns that string directly, if callable converts error to string to be returned.
+              Defaults to False.
+          handle_validation_error: Handle the content of the ValidationError thrown. If False
```

Comment:
nit - docstring misaligned

Reply:
also, i think the param description is a bit confusing (which is true for both the original description in the `BaseTool` class as well) - this is not handling the content of the error, but content of `ToolMessage` returned if the error is thrown